Client.xcodeproj/project.pbxproj

```diff
@@ -12992,7 +12972,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Manual;
 				COPY_PHASE_STRIP = NO;
-				CURRENT_PROJECT_VERSION = 76;
+				CURRENT_PROJECT_VERSION = "${inherited}";
```

Comment:
FYI: we shoud use `${inherited}` in all the targets and only provide the numeric version in the project file. This will keep a minimum and consitent changeset. We could also think of connecting this with the fastlane/VERSION file. But at the moment I am missing the imagination how this could work elegantly.

Reply:
Good point as a reminder for us 💡